port/unix/omrsock.c

```diff
@@ -76,8 +76,32 @@ omrsock_getaddrinfo_create_hints(struct OMRPortLibrary *portLibrary, omrsock_add
 
 int32_t
 omrsock_getaddrinfo(struct OMRPortLibrary *portLibrary, char *node, char *service, omrsock_addrinfo_t hints, omrsock_addrinfo_t result)
-{
-	return OMRPORT_ERROR_NOT_SUPPORTED_ON_THIS_PLATFORM;
+{	
+	if ((NULL == hints) || (NULL == result)) {
```

Comment:
It should be safe for hints to be null, in which case we need to pass a null hints object to the OS call.